resource/plugins/unreal/python/publisher/collectors/unreal_dependencies_publisher_collector.py

```diff
@@ -19,13 +19,38 @@ class UnrealLevelPublisherCollectorPlugin(
     plugin_name = 'unreal_dependencies_publisher_collector'
 
     def fetch(self, context_data=None, data=None, options=None):
-        '''Fetch all dependencies from the level'''
+        '''Fetch all dependencies from the level or the selected asset'''
 
-        level_asset_path = (
-            unreal.EditorLevelLibrary.get_editor_world().get_path_name()
+        if options.get('asset') is True:
```

Comment:
Maybe the option should be named selected_asset instead of asset or selection=True as other collectors